r2dbc-mysql/src/main/java/JasyncClientConnection.kt

```diff
@@ -21,6 +47,15 @@ class JasyncClientConnection(private val jasyncConnection: JasyncConnection) : C
         return executeVoid("COMMIT")
     }
 
+    override fun isAutoCommit(): Boolean {
+        return autoCommit
+    }
+
+    override fun setAutoCommit(autoCommit: Boolean): Publisher<Void> {
+        this.autoCommit = autoCommit
```

Comment:
Setting auto-commit state should be deferred (i.e. on completion or on subscribe of the `SET AUTOCOMMIT` statement)

Reply:
I changed it in latest commit, thanks!